src/pages/debrief/debrief.selector.ts

```diff
@@ -73,16 +75,14 @@ export const displayDrivingFaultComments = (data: TestData) : boolean => {
  *
  * Returns a container of array holding the propertyName and fullCompetencyLabel for each dangerous
  * fault recorded against a candidate.
- *
- * Is used to populate properties in dangerous-fault-comment.html for the entry of comments
- * relating to each dangerous fault.
  */
 export const getDangerousFaults = (faults: DangerousFaults): SeriousFaultsContainer[] => {
   const faultsEncountered: SeriousFaultsContainer[] = [];
```

Comment:
Don't like the naming of SeriousFaultscontainer - when used for getting dangerous faults. 
Probably my mistake in providing something call getSeriousOrDangerousFaults and not wanting to duplicate code, but would prefer a more generic name for this as it looks wrong that getDangerousFaults returns a SeriousFaultsContainer

Reply:
Yes I agree, `SeriousFaultsContainer` is from the original implementation, I'm hoping to rationalise that and `FaultComment` before merging. 👍

Reply:
+1 for ngSwitch. Seems like a good use case for it.